### PR TITLE
E-Sign updates 

### PR DESCRIPTION
**Pull request by eSignHQ**

### CODE REVIEW

File: certified-connectors/E-Sign/apiDefinition.swagger.json [hunk 1]
```diff
@@ -2,12 +2,12 @@
   "swagger": "2.0",
   "info": {
     "title": "E-Sign",
-    "description": "E-Sign Anything. Anywhere. It’s easy to e-sign and send your documents on any device with our secure electronic signature system.",
-    "version": "3.1",
+    "description": "E-Sign Anything. Anywhere. It\u2019s easy to e-sign and send your documents on any device with our secure electronic signature system.",
+    "version": "4.0",
```
Comment (v-EgorKozhadei) [inline certified-connectors/E-Sign/apiDefinition.swagger.json:6]: @eSignHQ are you sure that this version is correct? I think the version should be 5.0 for the update.

**Review by v-EgorKozhadei — COMMENTED** [review]
Hi @eSignHQ! Please check new comments, thank you!

**Comment by eSignHQ** [comment]
Updates have been added @v-EgorKozhadei

File: certified-connectors/E-Sign/apiDefinition.swagger.json [hunk 2]
```diff
@@ -2,12 +2,12 @@
...
     "title": "E-Sign",
-    "description": "E-Sign Anything. Anywhere. It’s easy to e-sign and send your documents on any device with our secure electronic signature system.",
-    "version": "3.1",
+    "description": "E-Sign Anything. Anywhere. It's easy to e-sign and send your documents on any device with our secure electronic signature system.",
+    "version": "5.0",
     "contact": {
-      "name": "E-Sign Support",
-      "url": "https://www.e-sign.co.uk",
-      "email": "support@e-sign.co.uk"
+        "name": "E-Sign Support",
```
Comment (v-EgorKozhadei) [inline certified-connectors/E-Sign/apiDefinition.swagger.json:8]: @eSignHQ can you please fix the indentation here?

Comment (v-EgorKozhadei) [inline certified-connectors/E-Sign/apiDefinition.swagger.json:8]: @eSignHQ can you please fix the indentation here?

Comment (eSignHQ) [reply certified-connectors/E-Sign/apiDefinition.swagger.json]: @v-EgorKozhadei fixed now

Comment (v-EgorKozhadei) [reply certified-connectors/E-Sign/apiDefinition.swagger.json]: @eSignHQ  There's still a difference in one whitespace between the existing version and the updated one. Can you please fix it one more time?

Comment (eSignHQ) [reply certified-connectors/E-Sign/apiDefinition.swagger.json]: @v-EgorKozhadei ok this should be ok now

**Review by v-EgorKozhadei — APPROVED** [review]

**Comment by v-EgorKozhadei** [comment]
Hello @eSignHQ ,
I hope you are doing well.

Congratulations, your pull request is approved and merged. Please [submit the files to ISV Studio](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) using your latest commit ID, please allow us up to 1-2 weeks to review your submission, and our Engineers will notify you in the “Activity Control” area in ISV Studio.

Please [Create an environment in the Preview region](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#create-an-environment-in-the-preview-region). You will use this environment later to [test your connector](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#test-your-connector) after Microsoft is done with the functional verification for your connector. 

We expect all tests to be completed within 1-2 weeks.

If your connector passes all tests, the deployment process begins, and it typically takes up to 3 to 4 weeks to deploy your connector to all of our regions.

Please let me know if you have any questions.

Thank you very much for working with us.

**Pull request merged**